store/policy.go

```diff
@@ -164,6 +164,24 @@ func (s *Store) GetBackupPlanPolicyByEnvID(ctx context.Context, environmentID in
 	return api.UnmarshalBackupPlanPolicy(policy.Payload)
 }
 
+// SomeDatabaseEnablesBackupPolicyInInstance returns whether at least one of the databases in the given instance enables backup policy.
+func (s *Store) SomeDatabaseEnablesBackupPolicyInInstance(ctx context.Context, instanceID int) (bool, error) {
```

Comment:
Should this be part of policy.go?

ContainsDatabaseWithBackup()

Reply:
Moved to store/backup.go

Reply:
Moved to store/backup.go